.vscode/settings.json

```
@@ -0,0 +1,3 @@
+{
+    "npm-scripts.showStartNotification": false
+}
```

```suggestion
}

```

É uma convenção (e boa prática) deixar uma linha em branco ao final de cada arquivo.

Dessa forma, ferramentas como o GitHub não exibem o ícone de "No newline at end of file".

---

E esta sugestão, não vai aplicar?